generators/client/templates/angular/package.json.ejs

```diff
@@ -143,6 +143,7 @@
     <%_ if (protractorTests) { _%>
     "webdriver-manager": "12.1.5",
     <%_ } _%>
+    "@openapitools/openapi-generator-cli": "0.0.9-3.3.4",
```

Comment:
Better to put 0.0.9-4.0.2

See https://github.com/jhipster/generator-jhipster/pull/9925

Reply:
updated to 0.0.14-4.0.2